cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/RunCommand.java

```diff
@@ -316,16 +316,16 @@ public String getName() {
 
     @Override
     public void printLongDesc(StringBuilder out) {
-        out.append("Run command runs a compiled Ballerina program. \n");
+        out.append("Run command executes a compiled Ballerina program. \n");
         out.append("\n");
         out.append("If a Ballerina source file or a module is given, \n");
-        out.append("run command compiles and runs it. \n");
+        out.append("the run command compiles and executes it. \n");
         out.append("\n");
         out.append("By default, 'ballerina run' executes the main function. \n");
         out.append("If the main function is not there, it executes services. \n");
         out.append("\n");
         out.append("If the -s flag is given, 'ballerina run' executes\n");
```

Comment:
I think this -s flag thing is outdated now.

Reply:
@MaryamZi

Reply:
Yes, we can remoe L327-328.

Reply:
Removed  L327-328.